ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/ConfigResource.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.server.resources;
+
+import io.confluent.ksql.rest.EndpointResponse;
+import io.confluent.ksql.rest.entity.ConfigResponse;
+import io.confluent.ksql.util.KsqlConfig;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ConfigResource {
+  private final Map<String, Object> allowedConfigs = new HashMap<>();
```

Comment:
Perhaps use `visibleConfigs` instead. Just to provide more meaning to what these configs are.